content/articles/How-to-start-up-with-Socket-Programming-with-Java/index.md

```diff
@@ -328,8 +268,6 @@ Closing connection
 ```
  Important note
```

Comment:
> Important note Port number may cause an error if the port is already in use. To fix this, change the port number to a value that is not commercially owned.

Add a \> in front to make it a blockquote. I mentioned this last time but I realized the way I said that may have been confusing, sorry. Let me know if you still don't understand what I mean here.